.changeset/chilly-olives-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/coin-hedera": patch
```

Comment:
We usually use `minor` now instead of `patch` for everything in LL and reserve `patch` for hotfixes